staging/src/k8s.io/kube-scheduler/config/v1beta1/types_pluginargs.go

```diff
@@ -90,6 +90,9 @@ type NodeResourcesFitArgs struct {
 	// A resource group name can't contain '/'.
 	// +listType=atomic
 	IgnoredResourceGroups []string `json:"ignoredResourceGroups,omitempty"`
+
```

Comment:
Note that the changes to this file will not be merged, we should revert them and add the type to v1beta2 once the other PR is merged.